src/app/list-of-components/page.tsx

```diff
@@ -0,0 +1,17 @@
+import SiteBanner from "@/components/banner/site-banner";
+import { DocsButton } from "@/components/buttons";
+import RootLayout from "../layout";
+
+
+export default function ListOfComponents() {
+  return (
+    
+      <RootLayout>
```

Comment:
NIT: I believe we can use the layout.tsx file to apply a layout to pages in this directory